### ISSUE TITLE
Why there is no camera timestamp in California dataset?

### ISSUE THREAD

**Issue by YiChenCityU** [weisongwen/UrbanLoco#3]
Hi,It's a great dataset. But I found that there is no camera data timestamp in California version. 
![Screenshot 2020-07-22 10:21:06](https://user-images.githubusercontent.com/12424523/88126969-40114d80-cc05-11ea-9692-84b29ef30117.png)

**Comment by weisongwen** [comment]
@YiChenCityU  Since there is no response after our email contact. We mark this issue as fixed. Thanks for your interests in our datasets.

**Comment by TakahashiJinxu** [comment]
Hello! I also found that there is no camera data timestamp in California version. Namely, when I tried to extract the camera and IMU data form the ROS bag using Kalibr (after a little modification to support compressed_imgmsg), I found the timestamps of all images are zero. The command I used was
kalibr_bagextractor --image-topics /camera_array/cam0/image_raw/compressed --imu-topics /imu_raw --output-folder dataset --bag ../CA-20190828194706_blur_align.bag
I also have tried UrbanNav-HK-Data20190428 in urbannav dataset and successfully got the correct timestamps when extracting the data by Kalibr. How can I get the timestamps of the images in California version in this dataset? Thank you in advance!
